docs/source/conf-datasources.rst

```diff
@@ -25,39 +25,28 @@ Example:
   {
     "data_sources": {
       "kinesis": {
-        "abc_corporate_stream_alert_kinesis": {
-          "logs": [
-            "box",
-            "pan"
-          ]
-        },
-        "abc_production_stream_stream_alert_kinesis": {
-          "logs": [
-            "inspec",
-            "osquery"
-          ]
-        }
-      },
+        "abc_corporate_stream_alert_kinesis": [
+          "box",
+          "pan"
+        ],
+        "abc_production_stream_stream_alert_kinesis": [
+          "inspec",
+          "osquery"
+        ]
       "s3": {
-        "abc.webserver.logs": {
-          "logs": [
+        "abc.webserver.logs": [
             "nginx"
-          ]
-        },
-        "abc.hids.logs": {
-          "logs": [
-            "carbonblack"
-          ]
-        }
+        ],
+        "abc.hids.logs": [
+          "carbonblack"
+        ]
       },
       "sns": {
-        "abc_sns_topic": {
-          "logs": [
-            "logstash"
-          ]
-        }
+        "abc_sns_topic": [
+          "logstash"
+        ]
       }
     }
   }
 
-Once datasources are defined, associated ``logs`` must have defined `schemas <conf-schemas.html>`_
+Once data sources are defined, associated ``logs`` must have defined `schemas <conf-schemas.html>`_
```

Comment:
this red is an indication that there is a formatting issue in your rst file. please check to see what is causing this (likely invalid json blob above)